al2/rust/hello-ddb/{{ cookiecutter.project_slug }}/README.md

```diff
@@ -0,0 +1,93 @@
+
+This project contains source code and supporting files for a serverless application that you can deploy with the SAM CLI. It includes the following files and folders:
+
+- `rust_app/Cargo.toml` - Project configuration file.
+- `rust_app/src/main.rs` - Code for the application's Lambda function.
+- `template.yaml` - A template that defines the application's AWS resources.
+
+The application uses several AWS resources, including Lambda functions and an API Gateway API. These resources are defined in the `template.yaml` file in this project. You can update the template to add AWS resources through the same deployment process that updates your application code.
+
+If you prefer to use an integrated development environment (IDE) to build and test your application, you can use the AWS Toolkit.  
+The AWS Toolkit is an open source plug-in for popular IDEs that uses the SAM CLI to build and deploy serverless applications on AWS. The AWS Toolkit also adds a simplified step-through debugging experience for Lambda function code. See the following links to get started.
+
+* [CLion](https://docs.aws.amazon.com/toolkit-for-jetbrains/latest/userguide/welcome.html)
+* [GoLand](https://docs.aws.amazon.com/toolkit-for-jetbrains/latest/userguide/welcome.html)
+* [IntelliJ](https://docs.aws.amazon.com/toolkit-for-jetbrains/latest/userguide/welcome.html)
+* [WebStorm](https://docs.aws.amazon.com/toolkit-for-jetbrains/latest/userguide/welcome.html)
+* [Rider](https://docs.aws.amazon.com/toolkit-for-jetbrains/latest/userguide/welcome.html)
+* [PhpStorm](https://docs.aws.amazon.com/toolkit-for-jetbrains/latest/userguide/welcome.html)
+* [PyCharm](https://docs.aws.amazon.com/toolkit-for-jetbrains/latest/userguide/welcome.html)
+* [RubyMine](https://docs.aws.amazon.com/toolkit-for-jetbrains/latest/userguide/welcome.html)
+* [DataGrip](https://docs.aws.amazon.com/toolkit-for-jetbrains/latest/userguide/welcome.html)
+* [VS Code](https://docs.aws.amazon.com/toolkit-for-vscode/latest/userguide/welcome.html)
+* [Visual Studio](https://docs.aws.amazon.com/toolkit-for-visual-studio/latest/user-guide/welcome.html)
+
+
+## Requirements
+* This template was tested with Rust v1.66.0 and above.
+
+## Deploy the sample application
+
+To deploy the application, you need the folllowing tools:
+
+* SAM CLI - [Install the SAM CLI](https://docs.aws.amazon.com/serverless-application-model/latest/developerguide/serverless-sam-cli-install.html)
+* Docker - [Install Docker community edition](https://hub.docker.com/search/?type=edition&offering=community)
+* [Rust](https://www.rust-lang.org/) version 1.64.0 or newer
+* [cargo-lambda](https://github.com/cargo-lambda/cargo-lambda) for cross-compilation
+
+To build and deploy your application for the first time, run the following in your shell:
+
+```bash
+sam build
+sam deploy
+```
+
+The first command will build the source of your application. The second command will package and deploy your application to AWS, with a series of prompts:
```

Comment:
Should we change this to mention that passing in `-g` for `sam deploy` will result in the prompts since we now have a default config? That way we still have this extra info here for those that want to read it.

Reply:
What about changing to something like "`sam deploy` will deploy the application with the default `samconfig.toml`. You can also run `sam deploy --guided`, which will display a series of prompts...`

Reply:
I like it